### PR TITLE
Arrow integration example

### PR DESCRIPTION
**Pull request by tampler**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -42,7 +46,7 @@ lazy val root =
       stdSettings("zio-arrow")
     )
     .settings(buildInfoSettings("zio-arrow"))
```
Comment (adamgfraser) [inline build.sbt]:
```suggestion
    .settings(buildInfoSettings("zio.arrow"))
```

Comment (adamgfraser) [reply build.sbt]: I think I made a mistake here. With this correction I think we should be able to keep the plugin enabled.

File: project/plugins.sbt [hunk 2]
```diff
@@ -6,5 +6,4 @@ addSbtPlugin("com.jsuereth"       % "sbt-pgp"       % "1.1.2")
 addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"  % "3.8.1")
 addSbtPlugin("org.scoverage"      % "sbt-scoverage" % "1.6.1")
 addSbtPlugin("org.scalameta"      % "sbt-mdoc"      % "2.1.1")
-addSbtPlugin("ch.epfl.scala"      % "sbt-bloop"     % "1.3.5")
```
Comment (adamgfraser) [inline project/plugins.sbt]: Do we want to get rid of this?

File: build.sbt [hunk 3]
```diff
@@ -23,6 +26,7 @@ inThisBuild(
 )
 
 ThisBuild / publishTo := sonatypePublishToBundle.value
+unmanagedResourceDirectories in (Compile, runMain) += baseDirectory(_ / "it").value
```
Comment (adamgfraser) [inline build.sbt]: Should we create a new `examples` project for examples of arrow usage?

File: src/it/src/scala/App0.scala [hunk 4]
```diff
@@ -0,0 +1,22 @@
+package zio.arrow.example
```
Comment (adamgfraser) [inline src/it/src/scala/App0.scala]: Great!

File: project/plugins.sbt (hunk 2 above)
Comment (tampler) [reply project/plugins.sbt]: My VSCode generates a separate file `metals.sbt` and tracks the version there. Just a wish for VSCode users

Comment (adamgfraser) [reply project/plugins.sbt]: Okay, I'm a VS Code user too so we're on the same page there.

@softinio Is this a change we should feed back into the Giter8 template or is there a reason we should keep this?

Comment (tampler) [reply project/plugins.sbt]: This is a very recent functionality. It started to add this file about week or two from now

File: build.sbt (hunk 3 above)
Comment (tampler) [reply build.sbt]: My plan is to add more and more integration examples and provide an application-driven real world development for this. 

I advice us to keep this for now. May change later

Comment (adamgfraser) [reply build.sbt]: That sounds fantastic!

File: project/plugins.sbt (hunk 2 above)
Comment (adamgfraser) [reply project/plugins.sbt]: Going to merge this now. @softinio If we are doing anything we shouldn't be here please let us know and we can fix subsequently.

**Pull request merged**